src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php

```diff
@@ -84,6 +84,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         if ($this->fileLinkFormatter) {
             $container = fn () => $this->getContainerBuilder($this->getApplication()->getKernel());
         }
+        if ($input->getOption('show-controllers')) {
+            trigger_deprecation('symfony/console', '7.3', 'Using --show-controllers is deprecated. They are shown by default now.');
```

Comment:
I'm not sure I see the point of this change. The arguments in the PR description look like personal preference to me. I'd prefer not changing this.

Reply:
In general yes, it's personal preference but there is also a technical argument to it. 

The TextDescriptor in List mode is the only thing that supports this flag. If you provide an argument for getting the route details or if you use a different formatter none of them use this flag. So this would also remove inconsistency in my opinion.

Reply:
I don't think that's sound enough to warrant a deprecation. Consistency is not the main aspect here.

Reply:
Fair enough. I'll remove it.